### PR TITLE
Add tests for higher than 2D cases

### PR DESCRIPTION
**Pull request by benjione**
Adding test for 3D (and higher), has to wait for multivariate PR to be accepted in ApproxFunBase.

### CODE REVIEW

File: test/MultivariateTest.jl [hunk 1]
```diff
@@ -12,6 +12,47 @@ using ApproxFunOrthogonalPolynomials: chebyshevtransform
...
+        end
+
+        @testset "3D" begin
+            f3 = Fun(Chebyshev()^3, [1.0])
+            @test f3(0.2, 0.4, 0.2) == 1.0
+        end
+
+        @testset "20D" begin
+            f20 = Fun(Chebyshev()^20, [1.0])
+            @test f20(rand(20)) == 1.0
```
[the remark this answers is not in the document]
Comment (benjione) [reply test/MultivariateTest.jl:52]: both should work, the `f20(rand(20)...)` will just be transformed to a vector afterwards.

**Comment by jishnub** [comment]
Could you bump the compat bound for `ApproxFunBase` to `0.7.40`? Tests added here would only pass with that as the minimum version.

**Comment by jishnub** [comment]
Could you also add an example of a non-trivial function in the evaluation section? This will ensure that the interlacing order is correct.

**Comment by benjione** [comment]
Sorry for the mess with the commits :smile: 

Sure, I will do that.

**Comment by jishnub** [comment]
Are you using `AbstractFFTs` for tests? It's not listed anywhere in `Project.toml` currently

**Pull request closed without merging**